### PR TITLE
{bio}[foss-2015b] PLINK 1.07

### PR DESCRIPTION
**Pull request by verdurin**
Ignore the branch name. It's version 1.07 (1.9 is still in beta).

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source_urls = ['http://pngu.mgh.harvard.edu/~purcell/plink/dist/']
+
+dependencies = [('zlib', '1.2.8')]
+
+# plink makefile is a little bit tricky so we pass every options as arguments
+# review plink original makefile for details
+# if you want "new version check" change to WITH_WEBCHECK="1", but if your compute nodes
+# have no internet access better leave it as is
+buildopts = ' CXX_UNIX="$CXX $CXXFLAGS" WITH_R_PLUGINS=1 WITH_WEBCHECK="" WITH_ZLIB=1 '
+buildopts += ' WITH_LAPACK=1 FORCE_DYNAMIC=1 LIB_LAPACK=$BLAS_LAPACK_LIB_DIR/libmkl_lapack.a'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb]: mkl in foss?

Comment (verdurin) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb]: Yes, this is the point where my ignorance of toolchain details rears itself again...

Will fix.

Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb]: the `FORCE_DYNAMIC` combined with specific a static LAPACK lib is also... interesting ;)

File: easybuild/easyconfigs/p/PLINK/PLINK-1.07_redeclaration.patch [hunk 2]
```diff
@@ -0,0 +1,68 @@
+diff --git a/sets.cpp b/sets.cpp
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07_redeclaration.patch]: please add a comment above this line what this patch is about, where you got it, and who created it (you?)
any text above the `diff` line is comments

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c5b943044d363a8726e9 for a full test report.

File: easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+patches = ['PLINK-1.07_redeclaration.patch']
+
+dependencies = [('zlib', '1.2.8')]
+
+# plink makefile is a little bit tricky so we pass every options as arguments
+# review plink original makefile for details
+# if you want "new version check" change to WITH_WEBCHECK="1", but if your compute nodes
+# have no internet access better leave it as is
+buildopts = ' CXX_UNIX="$CXX $CXXFLAGS" WITH_R_PLUGINS=1 WITH_WEBCHECK="" WITH_ZLIB=1 '
+buildopts += ' WITH_LAPACK=1 FORCE_DYNAMIC=1 LIB_LAPACK=$EBROOTOPENBLAS/lib/libopenblas.so'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-foss-2015b.eb]: don't hardcode `.so`, use `.../libopenblas.%s' % SHLIB_EXT` ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9002f6338e39dbf6496f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**